### ISSUE TITLE
UserWarning: Using a generator with `use_multiprocessing=True` and multiple workers may duplicate your data. Please consider using the`keras.utils.Sequence class. 

### ISSUE THREAD

**Issue by abdou31** [reigngt09/mask-rcnn#2]
Hello ,
 I get error when i tried to training balloon dataset from the first epoch ,
My laptop freeze and it back works just after rebooting

> UserWarning: Using a generator with `use_multiprocessing=True` and multiple workers may duplicate your data. Please consider using the`keras.utils.Sequence class. 

When i change use_multiprocessing=False and works=1 , the error appears again
How can i solve this error?

**Comment by Ovakefali13** [comment]
Same Problem here, did you find any help?

**Comment by 1nfinityLoop** [comment]
Just reset the runtime environment
